### PR TITLE
Pickle error message changed

### PR DESCRIPTION
**Pull request by twiecki**
CI is failing.

### CODE REVIEW

**Comment by twiecki** [comment]
pre-commit.ci autofix

**Comment by twiecki** [comment]
Ugh, seems like the strings are slightly different for python 3.11 and 3.12.

File: tests/test_config.py [hunk 1]
```diff
@@ -254,7 +254,10 @@ def test_config_pickling():
         configparser.IntParam(5, lambda i: i > 0),
         in_c_key=False,
     )
-    with pytest.raises(AttributeError, match="Can't pickle local object"):
+    with pytest.raises(
+        AttributeError,
+        match="Can't pickle local object 'test_config_pickling.<locals>.<lambda>'",
```
Comment (twiecki) [inline tests/test_config.py:259]:
```suggestion
        match="Can't (pickle|get) local object 'test_config_pickling.<locals>.<lambda>'",
```

**Comment by twiecki** [comment]
pre-commit.ci autofix

**Review by ricardoV94 — APPROVED** [review]

**Pull request merged**